### PR TITLE
Adding known issue for backup and restore feature

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/17650

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -788,3 +788,13 @@ When you install the gatekeeper operator on {ocp} version 4.9, the installation
...
+
+[#no-resource-status-after-restore]
+=== Application and policy show no resource status on managed cluster after a restore operation
+//2.4:17342
+
+When a restore operation is run on a new hub cluster, using data backed up from another hub cluster, the application and policy shows no status for resources on remote clusters. This happens because the search and policy add-ons are not reset to point to the new hub cluster. 
+
+From the new hub cluster, you must restart the `addon-certpolicyctrl` and `addon-search` for all managed clusters.
+
+//Does remote cluster mean offline cluster or local cluster?
```
Comment (dockerymick) [inline release_notes/known_issues.adoc]: @birsanv i left a comment here for you. When you have an opportunity, may you address it?

Comment (birsanv) [reply release_notes/known_issues.adoc]: @dockerymick remote clusters means managed clusters .. you can probably use the term managed clusters instead.

Also, should we comment that this issue was addressed in the 2.4.1 fixpack ?

**Review by birsanv — APPROVED** [review]
lgtm

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (dockerymick) [reply release_notes/known_issues.adoc]: @birsanv Thanks! If it was fixed in 2.4.1, i recommend not doc'ing this

**Comment by birsanv** [comment]
@dockerymick fyi this is the 2.4.1 task, completed but in verification for QE https://github.com/open-cluster-management/backlog/issues/17342

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (birsanv) [reply release_notes/known_issues.adoc]: fyi 2.4.1 task https://github.com/open-cluster-management/backlog/issues/17342

**Comment by dockerymick** [comment]
@birsanv Thanks! I will pause on this for now to see what QE says. If the fix is confirmed, I will close this PR and make a comment on our issue before closing it. Otherwise, I will merge the PR

**Comment by dockerymick** [comment]
@birsanv Since the dev issue is closed and the fix was verified in 2.4.1, it is best to not document the known issue. Do you have any thoughts or concerns? Otherwise, i will close this PR, comment and close the doc issue.

**Comment by birsanv** [comment]
unfortunately the app fix ( which is a search based issue ) did not go through so we need this readme until @xrajesh checks the fix in 2.4.2

@xrajesh can you please help validate this PR content

@dockerymick we should use the readme and remove any data related to policy ( remove any policy reference )

**Comment by xrajesh** [comment]
`From the new hub cluster, you must restart the `addon-certpolicyctrl` and `addon-search` for all managed clusters.` - Agree with this . @dockerymick @birsanv 
Question : Do we have to doc - Where from the hub cluster the user has to restart the pods ?

**Comment by birsanv** [comment]
@xrajesh you should explain how to restart the search pod ( remember I also had issues with that :) - go to the remote cluster, delete the ... pod
Mikela can review but you should provide the technical details

**Comment by dockerymick** [comment]
@xrajesh I attempted to add the commands to restart the pods. When you have an opportunity may you review this PR?

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -788,3 +788,17 @@ When you install the gatekeeper operator on {ocp} version 4.9, the installation
...
+
+When a restore operation is run on a new hub cluster, using data backed up from another hub cluster, the application and policy shows no status for resources on managed clusters. This happens because the search and policy add-ons are not reset to point to the new hub cluster. 
+
+From the new hub cluster, you must restart the `addon-certpolicyctrl` and `addon-search` for all managed clusters. Run the following commands to restart the pods:
+
+----
+oc delete po -n open-cluster-management addon-certpolicyctrl
+
+oc delete po -n open-cluster-management addon-search
+----
```
Comment (xrajesh) [inline release_notes/known_issues.adoc:802]: @dockerymick @birsanv This pods in the managed clusters are below - also they run in the namespace open-cluster-management-agent-addon 
```
klusterlet-addon-appmgr-8fc4566c8-cqsj9                      1/1     Running   18 (7h24m ago)   12d
klusterlet-addon-certpolicyctrl-584bb685d6-qscf6             1/1     Running   9 (7h24m ago)    12d
klusterlet-addon-iampolicyctrl-79b9796cdd-ljcp7              1/1     Running   8                12d
klusterlet-addon-operator-6c9b96497f-dcmmt                   1/1     Running   8                12d
klusterlet-addon-policyctrl-config-policy-748c766886-lgmrp   1/1     Running   8                12d
klusterlet-addon-policyctrl-framework-8544b6cd47-d6hz4       3/3     Running   52 (7h24m ago)   12d
klusterlet-addon-search-7ddc8f6967-lwd8x                     1/1     Running   8                12d
klusterlet-addon-workmgr-5994c45b6f-67lqf                    1/1     Running   27 (7h24m ago)   12d
```
Should we specifically say to delete search pod like this 
`oc get pods -n open-cluster-management-agent-addon  | grep search | awk '{print $1}' | xargs kubectl delete pod `

Comment (dockerymick) [reply release_notes/known_issues.adoc:802]: @xrajesh thanks! just updated the command

**Pull request merged**